### ISSUE TITLE
Travis build fails on OSX+Accelerate

### ISSUE THREAD

**Issue by vladimir-ch** [gonum/lapack#94]
I cannot test it by myself right now but it seems like LAPACKE in not available in the Accelerate framework.

**Comment by jonlawlor** [comment]
Yes, that is what I thought from the logs as well.  I'm going to give up on Accelerate and try to get OpenBLAS working for gonum/blas, and then try that with lapack.

**Comment by jonlawlor** [comment]
That being said, it looks like there are lapack libs in the osx environment:

https://travis-ci.org/gonum/lapack/jobs/108044692#L126
https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man7/LAPACK.7.html

My macbook is back in action so I'll fiddle with it and see if there is a way to get it to link.

**Comment by vladimir-ch** [comment]
That's CLAPACK, not LAPACKE, unfortunately. From my brief googling it seems that LAPACKE is not in Accelerate.

**Comment by jonlawlor** [comment]
Do you think it would be possible, and useful, to make from the reference lapacke and link against Accelerate's BLAS?

**Comment by btracey** [comment]
Is there a resolution to this?
